### ISSUE TITLE
fal installation instructions with dbt 1.0

### ISSUE THREAD

**Issue by thomasaarholt** [fal-ai/fal#87]
Trying fal now after a recommendation through Slack. I installed dbt 1.0 by `pip install dbt-core`, as well as fal by `pip install fal`. 
I ran into a few snags when installing fal in a clean python 3.9 environment, and it might be worth adding a note about dbt 1.0 to the fal readme.

- fal currently doesn't install dbt. I assume this is because of the change from `dbt` to `dbt-core`. 
- fal relies on bigquery [here](https://github.com/fal-ai/fal/blob/9622f85a46e2d58858e7128ed396524a25f4c8ef/src/faldbt/project.py#L289), which means that one needs to run `pip install dbt-bigquery`. This should be explicitly stated on the readme, and preferably not be a requirement.

**Comment by thomasaarholt** [comment]
For the second point, it would be much better to use:

```
from dbt.adapters.factory import load_plugin
load_plugin("bigquery") # returns dbt.adapters.bigquery.connections.BigQueryCredentials
```

**Comment by mkam039** [comment]
Hi @thomasaarholt! Thanks for raising this issue. We are actively working on supporting dbt 1.0 (see #69) and should release the new version soon. You're right, we should add a note about dbt 1.0 in README.

**Comment by thomasaarholt** [comment]
Ah! I searched for `1.0` and didn't find the PR (I had to type 1.0.0...) :)

**Comment by thomasaarholt** [comment]
My bigquery fix above works on 0.20 as well, I just checked:
![image](https://user-images.githubusercontent.com/2721423/147217991-942e2d41-a972-41c4-82bb-649e8faa634f.png)

**Comment by thomasaarholt** [comment]
Should I make a PR?

**Comment by thomasaarholt** [comment]
Ah, I see you have a fix as well in the 1.0.0 PR :) I'll leave that to you!

**Comment by chamini2** [comment]
@thomasaarholt, we have just released a version compatible with dbt 1.0.0.

```
pip install fal==0.1.35
```

Please let us know how it goes and if anything else comes up!
